### PR TITLE
Fix old Django version detection

### PR DESCRIPTION
**Pull request by CaseyFaist**
Running tests from privileged account for #923

### CODE REVIEW

File: bin/steps/pip-install [hunk 1]
```diff
@@ -38,10 +38,10 @@ if [ ! "$SKIP_PIP_INSTALL" ]; then
         mcount "failure.none-version"
     fi
 
-    if grep -q 'django==1.*' requirements.txt; then
+    if grep -qi '^django==1.*' requirements.txt; then
```
Comment (hone) [inline bin/steps/pip-install]: Does `requirements.txt` accept case insensitive filenames?

Comment (danielleadams) [inline bin/steps/pip-install]: I believe this is the "ignore case" flag, right? Is there a reason this needs to be added?

Comment (hone) [reply bin/steps/pip-install]: From talking with @CaseyFaist, here's the [PEP](https://www.python.org/dev/peps/pep-0426/#name).

Comment (CaseyFaist) [reply bin/steps/pip-install]: pip install is case insensitive 👍

**Pull request merged**